content/includes/header.html

```diff
@@ -47,6 +47,7 @@
 <meta name="msapplication-TileColor" content="#144073">
 
 <!-- CSS -->
+<link rel="stylesheet" href="/generated/style.css">
```

Comment:
Yeah, I think that's a great next step. I think it'd be tough to figure out _how_ to split them all up without breaking things, but after we get visual regression testing, it'd be easier to test _if_ things are broken, anyhow.

Anybody know if there's a tool to find unused styles in a stylesheet? I'll do some poking around, but that seems like a kind of hairy problem to solve...